src/controllers/confirmation.controller.ts

```diff
@@ -92,10 +92,21 @@ const route = async (req: Request, res: Response, next: NextFunction): Promise<v
       return next(new Error("No new filing due date returned by the PTF API"));
     }
 
+    let overdueFiling: string;
+    if (companyProfile.isAccountsOverdue && !companyProfile.isConfirmationStatementOverdue) {
```

Comment:
how about:
`let overdueFiling: string;
    if (companyProfile.isAccountsOverdue) {
      overdueFiling = "accounts";
      if (companyProfile.isConfirmationStatementOverdue) {
        overdueFiling = overdueFiling.concat(" and confirmation statement");
      }
    } else {
      // TODO Handle scenario when only confirmation statement is overdue LFA-1479
      overdueFiling = "UNDEFINED";
    }`